src/main/java/pl/multiplex/multiplexreservation/MultiplexReservationApplication.java

```diff
@@ -1,13 +1,153 @@
     }
 
+    @Bean
+    CommandLineRunner commandLineRunner(
+            ReservationRepo reservationRepo,
+            MovieRepo movieRepo,
+            CinemaRoomRepo cinemaRoomRepo,
+            ScreeningRepo screeningRepo,
+            SeatRepo seatRepo,
+            UserRepo userRepo) {
+        return args -> {
+
+            Set<Screening> screenings= new HashSet<Screening>();
+            Set<Movie> movies= new HashSet<Movie>();
+            Set<CinemaRoom> cinemaRooms= new HashSet<CinemaRoom>();
+            Set<Reservation> reservations= new HashSet<Reservation>();
+            Set<User> users= new HashSet<User>();
+            Set<Seat> seats= new HashSet<Seat>();
+
+            movies = generateMovies(3);
+            cinemaRooms = generateCinemaRooms(3,2,3,seatRepo);
+            screenings = generateScreenings(3,cinemaRooms,movies);
+            User admin = new User("Admin","Admin","Admin");
+            reservations = generateReservations(admin,screenings,0.3,reservationRepo,userRepo);
+
+            movieRepo.saveAll(movies);
+            cinemaRoomRepo.saveAll(cinemaRooms);
+            screeningRepo.saveAll(screenings);
+            reservationRepo.saveAll(reservations);
+        };
+    }
+
+    public Set<Movie> generateMovies(int nrOfMovies) {
+        Set<Movie> movies = new HashSet<Movie>();
+        Set<Screening> screenings = new HashSet<Screening>();           //to add
+        String[] movies_names = new String[3];
+        double[] durations = new double[3];
+        for(int i =0;i<nrOfMovies;i++) {
+            movies_names[i] = "Movie nr " +(i+1);
+            if(i%2 ==0) {
+                durations[i] = 2 + ((double)i/10); } else {
+                durations[i] = 2 - ((double)i/10);
+            }
+            movies.add(new Movie(movies_names[i],durations[i],screenings));
+        }
+        return movies;
+    }
+
+    public Set<Seat> generateSeats(int nrOfRows,int nrOfColumns){
+        Set<Reservation> reservations = new HashSet<Reservation>();                   //to add
+        Set<Seat> seats =  new HashSet<Seat>();
+        for(int j =0;j<nrOfRows;j++) {
+            for (int k = 0; k < nrOfColumns; k++) {
+                seats.add(new Seat(j + 1, k + 1, null, reservations));
+            }
+        }
+        return  seats;
+    }
+
+    public Set<CinemaRoom> generateCinemaRooms(int nrOfRooms,int nrOfRows,int nrOfColumns,SeatRepo seatRepo){
+        Set<Screening> screenings = new HashSet<Screening>();           //to add
+        Set<CinemaRoom> cinemaRooms = new HashSet<CinemaRoom>();
+        for(int i =0;i<nrOfRooms;i++) {
+            Set<Seat> seats = null;
+            seats = generateSeats(nrOfRows, nrOfColumns);
+            seatRepo.saveAll(seats);
+            CinemaRoom room = new CinemaRoom(screenings,seats);
+            cinemaRooms.add(room);
+            Iterator<Seat> seatIter = seats.iterator();
+            while(seatIter.hasNext()){
```

Comment:
Takiej konstrukcji nie używa się od ~Javy 6 - teraz mamy Javę 16 :) Masz pomysł jak to napisać inaczej?

Reply:
Znalazłem takie rozwiązanie w internecie i automatycznie stałem się jego fanem, ale skoro są jakieś nowsze i lepsze to pewnie bym je jeszcze bardziej polubił .

Reply:
"Nowsze" to mało powiedziane, Java 6 jest z 2006 roku :)